docs/source/tracking.rst

```diff
@@ -218,11 +218,19 @@ Here is an example plot of the :ref:`quick start tutorial <quickstart>` with the
   X-axis relative time - graphs the time relative to the first metric logged, for each run
 
 
+.. _automatic-logging:
+
 Automatic Logging
 =================
 
-Automatic logging allows you to log metrics, parameters, and models without the need for explicit
-log statements, and is currently supported for:
+Automatic logging allows you to log metrics, parameters, and models without the need for explicit log statements.
+
+There are two ways to use autologging:
+
+#. Call :py:func:`mlflow.autolog` before your training code. This will enable autologging for every supported library you have installed as soon as you import it.
```

Comment:
```suggestion
#. Call :py:func:`mlflow.autolog` before your training code. This will enable autologging for each supported library you have installed as soon as you import it.
```